### PR TITLE
Minor DAO ui fixes

### PR DESCRIPTION
**Pull request by ripcurlx**

### CODE REVIEW

File: desktop/src/main/java/bisq/desktop/components/MenuItem.java [hunk 1]
```diff
@@ -58,7 +58,7 @@ public MenuItem(Navigation navigation,
         this.viewClass = viewClass;
         this.baseNavPath = baseNavPath;
 
-        setLabelText(title);
+        setLabelText(title.toUpperCase());
```
Comment (sqrrm) [inline desktop/src/main/java/bisq/desktop/components/MenuItem.java]: This doesn't handle the 'Vote on proposals'/'Browse open proposals' text that changes depending on phase

**Review by sqrrm — CHANGES_REQUESTED** [review]
NACK

Need to change all labels to upper case

File: desktop/src/main/java/bisq/desktop/components/MenuItem.java (hunk 1 above)
Comment (sqrrm) [reply desktop/src/main/java/bisq/desktop/components/MenuItem.java]: Could be fixed in GovernanceView.java:90/92

Comment (ripcurlx) [reply desktop/src/main/java/bisq/desktop/components/MenuItem.java]: Thanks for pointing it out. Just fixed it by converting to uppercase also on label update.

Comment (ripcurlx) [reply desktop/src/main/java/bisq/desktop/components/MenuItem.java]: And I removed now an unnecessary uppercasing. Should have looked through the code more carefully 🙄

**Review by sqrrm — APPROVED** [review]
ACK

Of course, much better to handle it there :)

File: desktop/src/main/java/bisq/desktop/components/MenuItem.java (hunk 1 above)
Comment (sqrrm) [reply desktop/src/main/java/bisq/desktop/components/MenuItem.java]: Yes, better to only set it inside setLabelText as you do now

**Pull request merged**